### PR TITLE
Default Contiki entry point

### PR DESCRIPTION
**Pull request by mtusnio**
A default makefile + test main.c file to use as the entry point when creating a Contiki application.

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -43,17 +43,39 @@
...
+#include "sys/clock.h"
+#include "dev/leds.h"
+
 PROCESS(main_process, "Main process");
 AUTOSTART_PROCESSES(&main_process);
 /*---------------------------------------------------------------------------*/
 PROCESS_THREAD(main_process, ev, data)
 {
-  PROCESS_BEGIN();
+    PROCESS_BEGIN();
```
Comment (pratik-prajapati-imgtec) [inline src/main.c]: Contiki's style is to give two spaces for indentation.
Please replace tabs with two spaces in all PR's.

**Comment by mtusnio** [comment]
Fixed indentation.

**Comment by manohar-narkhede-imgtec** [comment]
The basic directory structure of Contiki is getting changed. The application can be placed at https://github.com/Creatordev/contiki/tree/3.0-mikro-e/platform/mikro-e/apps.

**Comment by mtusnio** [comment]
Renamed to sample-app and moved to apps.

**Comment by manohar-narkhede-imgtec** [comment]
Can you please re-base and squash into one?

**Comment by mtusnio** [comment]
Done

**Comment by manohar-narkhede-imgtec** [comment]
Approved.

**Pull request merged**